packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -64,9 +64,27 @@ The third generation of the Okta Sign-In Widget doesn’t guarantee the stabilit
 
 * If you use an Okta-managed TLS certificate, you don't need a [Certificate Authority Authorization (CAA)](https://datatracker.ietf.org/doc/html/rfc6844) record. However, if you do have a CAA record, keep the following in mind:
 
-    * If it's your first time setting up a custom domain with an Okta-managed certificate, you need to add `letsencrypt.org` to the issuers list or Okta can't get the TLS certificate. See [Let's Encrypt - Using CAA](https://letsencrypt.org/docs/caa/).
+  * If it's your first time setting up a custom domain with an Okta-managed certificate, you need to add `letsencrypt.org` to the issuers list or Okta can't get the TLS certificate. See [Let's Encrypt - Using CAA](https://letsencrypt.org/docs/caa/).
 
-    * If you have an Okta-managed certificate and you later get a CAA record, Okta can't renew your certificate. You must either add `letsencrypt.org` to the issuers list or remove the CAA record.
+  * If you have an Okta-managed certificate and you later get a CAA record, Okta can't renew your certificate. You must either add `letsencrypt.org` to the issuers list or remove the CAA record.
+
+* If you use your own TLS certificate, keep in mind the following:
```

Comment:
would 'consider' be easier for non-english speakers to understand than 'keep in mind', which might be a phrase that might be more difficult to parse?

Reply:
Good call. I'll change it and in one other place, too.